docs/getting-started/GETTING_STARTED.md

```diff
@@ -392,3 +392,41 @@ or
 
 In all scenarios, even if the user gives a non-temporary table, only temporary
 tables will actually be deleted.
+
+## Tying Astro SDK decorators to traditional Airflow Operators
```

Comment:
Is the getting started tutorial the best place for this documentation?

I feel it could be best placed within our Sphinx documentation as a reference page. I believe the idea of the tutorial was to walk users through a first example of the Python SDK without introducing too many possibilities. Could you confirm this, @mikeshwe ?

Reply:
@tatiana this is one of the most common questions we get asked (how can I use astro-sdk with traditional airflow operators) so it seemed like the getting started page was a good place to do it. Glad to discuss other options though.

Reply:
I think this is fair. 👍